libs/sdk-ui-dashboard/src/dashboard/Dashboard.tsx

```diff
@@ -184,7 +187,7 @@ const DashboardLoading: React.FC<IDashboardProps> = (props: IDashboardProps) =>
 
     useEffect(() => {
         if (!loading && result === undefined) {
-            dispatch(loadDashboard(props.config, props.permissions));
+            dispatch(loadDashboard(props.config, props.permissions, "initialLoad"));
```

Comment:
The "initialLoad" should be exported from the package so that the users can safely use it in their application.

Reply:
fixed